docs/application/dotnet/guides/connectivity/webview.md

```diff
@@ -0,0 +1,165 @@
+# WebView
+
+You can access Web pages and Web content in your application. You can also use various features for Web browsing, such as loading and displaying Web pages and navigating through the browsing history.
```

Comment:
@younghahung
As already noted in the previous comment by @mijin85cho it would be good if we add more detail to the overview. 

@mijin85cho @younghajung
I suggest the following, 
Can you please give your viewpoint: 

The WebView allows you to display web content on the web pages. It does not include a full-fledged web browser that essentially has an address bar and other navigation controls. All that WebView does primarily is to display a web page and its web browsing feature enables you to load, display web pages and navigate through the browsing history.

Reply:
I think that most developers recognize that WebView does not directly support a URL bar or navigation bar. So I think that current overview is enough.

Reply:
@younghajung , then it is fine.